### ISSUE TITLE
[Question] есть ли отличие пересылки сообщений без авторства оригиальной телеги от вашей?

### ISSUE THREAD

**Issue by yo8z6gv** [kotatogram/kotatogram-desktop#233]

**Comment by ilya-fedin** [comment]
Да, оригинальная использует API-запрос, котатовская берет контент сообщения и копирует его

**Comment by yo8z6gv** [comment]
> использует специальный API-запрос

мог бы скинуть хоть ссылку где подробнее можно чекнуть

**Comment by ilya-fedin** [comment]
Не очень понимаю, что ты хочешь чекнуть...

**Comment by RadRussianRus** [comment]
Если речь про то, какой именно метод, то вот документация по нему:
https://corefork.telegram.org/method/messages.forwardMessages

Технически, это тот же самый метод, что был и для обычной пересылки, просто туда добавили флаги `drop_author` и `drop_media_captions`.

**Comment by yo8z6gv** [comment]
> Если речь про то, какой именно метод, то вот документация по нему: https://corefork.telegram.org/method/messages.forwardMessages
> 
> Технически, это тот же самый метод, что был и для обычной пересылки, просто туда добавили флаги `drop_author` и `drop_media_captions`.

это намного лучше ответ, но если совсем точно, то вот конкретная ситуация:
![2021-10-06 17_02_54-Kotatogram (704118)](https://user-images.githubusercontent.com/62700253/136218415-f57fd10b-ec78-4613-9be2-fc4f8842c332.jpg)

если пересылать через "копирование", метод который в telegramX и вашем клиенте; такого не встретить. может ли подобное произойти с материалами пересланными с drop_author и drop_media_captions?

**Comment by ilya-fedin** [comment]
Теоретически - да
Практически - кто знает?
